output-worker/src/main.rs

```diff
@@ -72,6 +72,10 @@ struct Config {
     /// Timeout for obtaining a HTTP response from the target, including connect phase (if exceeded, request attempt will fail)
     #[clap(long, env, value_parser = humantime::parse_duration, default_value = "15s")]
     timeout: Duration,
+
+    /// Signature Header name
+    #[clap(long, env, default_value = "X-Hook0-Signature")]
+    signature_header_name: String,
```

Comment:
I believe you could ask for a `HeaderName` here instead of a `String`

Reply:
Cool, that's much neater! This is my first time using clap, or Rust for that matter...